src/core/src/op/paged_attention.cpp

```diff
@@ -147,40 +147,53 @@ void PagedAttentionExtension::validate_and_infer_types() {
                           get_input_element_type(12),
                           ".");
 
-    if (get_input_size() == 16) {
```

Comment:
is it possible to use allow_score_aggregation and allow_cache_rotation separately?

theoretically, using the different modes may lead to the same number or inputs
17 inputs
           bool allow_score_aggregation = true
           bool allow_cache_rotation = true

16 inputs:
           bool allow_score_aggregation = true
           bool allow_cache_rotation = false

16 inputs:
           bool allow_score_aggregation = false
           bool allow_cache_rotation = true
           
 how to differentiate the 16inputs cases?

Reply:
The last 3 (rotation related) inputs should either all be present, or none of them should. Due to the way the arguments are handled top-to-bottom I see no way to express this explicitly in code